crates/l2/contracts/src/l1/interfaces/ICommonBridge.sol

```diff
@@ -6,26 +6,21 @@ pragma solidity =0.8.29;
 /// @notice A CommonBridge contract is a contract that allows L1<->L2 communication
 /// from L1. It both sends messages from L1 to L2 and receives messages from L2.
 interface ICommonBridge {
-    /// @notice A deposit to L2 has initiated.
-    /// @dev Event emitted when a deposit is initiated.
-    /// @param amount the amount of tokens being deposited.
-    /// @param to the address that will be called in the L2.
-    /// @param depositId Id used to differentiate deposits with same amount and recipient.
-    /// @param from the address that initiated the deposit.
+    /// @notice A privileged transaction to L2 has initiated.
+    /// @dev Event emitted when a privileged transaction is initiated.
+    /// @param from the address that initiated the transaction.
+    /// @param to the recipient on L2
+    /// @param transactionId Id used to make transactions unique
+    /// @param value the value of the transaction
     /// @param gasLimit the gas limit for the deposit transaction.
     /// @param data The calldata of the deposit transaction.
-    /// @param l2MintTxHash the hash of the transaction that will finalize the
-    /// deposit in L2. Could be used to track the status of the deposit finalization
-    /// on L2. You can use this hash to retrive the tx data.
-    /// It is the result of keccak(abi.encode(transaction)).
-    event L1ToL2Message(
-        uint256 indexed amount,
+    event PrivilegedTxSent (
+        address indexed from,
```

Comment:
I discussed this with @ilitteri and @jrchatruc, and we agreed that we should add an address indexed L1From (the name can be adjusted) so that wallets can track their own privileged transactions for UX purposes.

We also agreed that this should be the only indexed parameter, unless someone can provide a strong reason to keep from, to, or id indexed.

L1From should be set to msg.sender when emitted.

Reply:
Can't `from` be used for this purpose?